examples/plot_icp_coregistration.py

```diff
@@ -1,6 +1,6 @@
 """
-Iterative Closest Point (ICP) coregistration.
-=============================================
+Iterative Closest Point coregistration
```

Comment:
Why did you remove "ICP"? Since this abbreviation is used later (e.g. L64), it's good if it is clearly defined once, no?

Reply:
Those heading appear as titles in the "Example gallery" pages. The one with "ICP" acronym in parenthesis and the one with "(2011)" after "Nuth & Kaab coregistration" stuck out because of it.

I moved the acronym further below the text, and added the references for ICP to solve #256